monai/apps/auto3d/data_analyzer.py

```diff
@@ -0,0 +1,541 @@
+            # 'percentile_00_5' : partial(torch.quantile, q=.005),
+            # 'percentile_99_5' : partial(torch.quantile, q=.995),
+            # 'percentile_10_0' : partial(torch.quantile, q=.10),
+            # 'percentile_90_0' : partial(torch.quantile, q=.90),
+            "stdev": partial(torch.std, unbiased=False),
+        }
+        # allow mapping the output of self.operations to new keys (save computation time)
+        # the output from torch.quantile is mapped to four keys.
+        self.operations_mappingkey = {
+            "percentile": ["percentile_00_5", "percentile_10_0", "percentile_90_0", "percentile_99_5"]
+        }
+        # np version of self.operations. torch operation has inconsistent interfaces.
+        # only used in the summary part.
+        self.operations_np = {
+            "max": np.max,
+            "mean": np.mean,
+            "median": np.median,
+            "min": np.min,
+            "percentile": partial(np.quantile, q=[0.005, 0.10, 0.90, 0.995]),
+            # 'percentile_00_5' : partial(np.percentile, q=0.5),
+            # 'percentile_99_5' : partial(np.percentile, q=99.5),
+            # 'percentile_10_0' : partial(np.percentile, q=10),
+            # 'percentile_90_0' : partial(np.percentile, q=90),
+            "stdev": np.std,
+        }
+        # define summary functions for output from self.operations. For exmaple,
+        # how to combine max intensity from each case (image)
+        self.operations_summary = {
+            "max": np.max,
+            "mean": np.mean,
+            "median": np.mean,
+            "min": np.min,
+            "percentile_00_5": np.mean,
+            "percentile_99_5": np.mean,
+            "percentile_10_0": np.mean,
+            "percentile_90_0": np.mean,
+            "stdev": np.mean,
+        }
+
+    def _get_case_image_stats(self):
+        ###############################################################
+        """Generate case_stats"""
+        # retrieve transformed data from self.data
+        start = time.time()
+        ndas = self.data["image"]
+        ndas = [ndas[i] for i in range(ndas.shape[0])]
+        if "nda_croppeds" not in self.data.keys():
+            self.data["nda_croppeds"] = [self._get_foreground_image(_) for _ in ndas]
+        nda_croppeds = self.data["nda_croppeds"]
+        # perform calculation
+        case_stats = {
+            "image_stats": {
+                "shape": [list(_.shape) for _ in ndas],
+                "channels": len(ndas),
+                "cropped_shape": [list(_.shape) for _ in nda_croppeds],
+                # 'spacing': np.tile(np.array(self.data['image_meta_dict']['pixdim'][1:4]), [len(ndas),1]).tolist(),
+                "spacing": np.tile(np.diag(self.data["image_meta_dict"]["affine"])[:3], [len(ndas), 1]).tolist(),
+                "intensity": [self._stats_opt(_) for _ in nda_croppeds],
+            }
+        }
+        logger.debug(f"Get image stats spent {time.time()-start}")
```

Comment:
Missed blanks before and after "-": `{time.time() - start}`

Reply:
Noted and fixed

Reply:
Fixed in a1055d826e8fb1760d2da77f4f5612e4d1d104cb (if not then next commit)